modules/library/main/src/main/java/org/geotools/styling/visitor/Measure.java

```diff
@@ -2,7 +2,7 @@
  *    GeoTools - The Open Source Java GIS Toolkit
  *    http://geotools.org
  *
- *    (C) 2013, Open Source Geospatial Foundation (OSGeo)
+ *    (C) 2013-2016, Open Source Geospatial Foundation (OSGeo)
```

Comment:
Thanks! We have just recently relaxed our stance on updating headers - no harm in changing though